rust/README.md

```diff
@@ -51,11 +72,11 @@ This populates data in two git submodules:
 Create two new environment variables to point to these directories as follows:
 
 ```bash
-export PARQUET_TEST_DATA=/path/to/arrow/cpp/submodules/parquet-testing/data
-export ARROW_TEST_DATA=/path/to/arrow/testing/data/
+export PARQUET_TEST_DATA=../../cpp/submodules/parquet-testing/data
```

Comment:
Relative paths potentially won't work when running `cargo test` from within the crate directories themselves rather than from the workspace directory. Perhaps we could do this instead?

```
export PARQUET_TEST_DATA=`pwd`/../../cpp/submodules/parquet-testing/data
export ARROW_TEST_DATA=`pwd`/../../testing/data/
```

I'm assuming this works both on Mac and Linux, at least.

Reply:
You are right, good catch. I have fixed it.